src/Toggle/Entity/ActivatableToggle.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Clearbooks\Labs\Toggle\Entity;
+
+interface ActivatableToggle
```

Comment:
@Clearbooks-Tom @craigjbass 
Right now the IsToggleActive gets the already decided answers (userToggles) from the ActivatableToggleGateway
If ActivatableToggleGateway contains the user activated toggles (userToggles)

My version:
then we have to make sure:
1) disabled visible toggles (releaseToggles) deactivate user activated toggles (userToggles)

## Who can activate a userToggle:
1) staff member activates a releaseToggle (auto subscription to releaseToggles triggers userToggle)
2) user action (only releaseToggle can be a userToggle)
3) group action (switches userToggles on multiple accounts)
4) admin/support on any account (only releaseToggles)

## Who can deactivate a userToggle:
1) staff member revoke releaseToggles status (remove all userToggle)
2) user deactivates userToggle
3) group action (switches userToggles on related accounts)
4) admin/support on any account

It should be stored! 

Any thoughts on this?